fitbenchmarking/fitbenchmarking
# Adding SciPy minimize routines

#### Description of Work

Fixes #523 

This PR adds the SciPy minimizer routines that do not require Hessian information. 

#### Testing Instructions

1. Check test pass and implementation is clear.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
